### ISSUE TITLE
[Bug]: IndexOutOfBoundsException after ping with at least 2 players on a server

### ISSUE THREAD

**Issue by inkisu** [LukenSkyne/Minecraft-Ping-Wheel#28]
### Preflight Checklist

- [X] I am using the [latest version](https://github.com/LukenSkyne/Minecraft-Ping-Wheel/releases) of Ping Wheel for the corresponding mod loader.
- [X] The server I am playing on has Ping Wheel installed.
- [X] I have searched the [issue tracker](https://github.com/LukenSkyne/Minecraft-Ping-Wheel/issues?q=is%3Aissue) for a bug report that matches the one I want to file, without success.

### Which mod loaders have you experienced the issue with?

Fabric

### What versions of Minecraft have you experienced the issue in?

1.20.2

### Expected Behavior

I would expect the client not to crash.

### Actual Behavior

When I am alone on the server, the ping wheel works as expected. 
When I try to ping with at least 2 players on the server, the server throws an exception and the client crashes / disconnects.

Error message:
```
[11:07:47] [Netty Epoll Server IO #1/ERROR]: Encountered exception while handling in channel with name "ping-wheel-c2s:ping-location"
java.lang.IndexOutOfBoundsException: readerIndex(75) + length(1) exceeds writerIndex(75): UnpooledByteBufAllocator$InstrumentedUnpooledUnsafeHeapByteBuf(ridx: 75, widx: 75, cap: 256)
```

### Logs URL

https://mclo.gs/R5eKsCn

### Additional Information

Initially, I had [FerriteCore](https://modrinth.com/mod/ferrite-core) and [Memory Leak Fix](https://modrinth.com/mod/memoryleakfix) installed. I removed them to narrow out compatibilty issues.

The issue persists after removing both mods.

### Edit:
I set up a minimal example to recreate the issue. 
#### Server
- Java 17
- Minecraft 1.20.2
- fabricloader 0.14.24
- fabric-api 0.90.7+1.20.2
- carpet 1.4.121+v231011
- ping-wheel 1.6.0

#### Client
- Java 17 
- Minecraft 1.20.2
- fabricloader 0.14.24
- fabric-api 0.90.7+1.20.2
- ping-wheel 1.6.0

After adding a carpet bot with `/player Steve spawn` the same error is thrown after pinging.  

#### Server Logs (minimal example)

https://mclo.gs/HIPygCm

**Comment by LukenSkyne** [comment]
Thanks for reporting!
This was an oversight when migrating to 1.20.2. Next version will fix this problem.

**Comment by LukenSkyne** [comment]
Fixed in v1.7.0
